src/styles/utils/mixins.scss

```diff
@@ -0,0 +1,43 @@
+@mixin onHover() {
+  transition: transform, 0.5s;
```

Comment:
`transition` should set to &__day, not to hover state, to make it works correctly on moving mouse away from cell